.mk/k8s.mk

```diff
@@ -17,13 +17,17 @@ K8S_CONF_DIR = k8s/conf
 CLUSTER_CONFIG_ROLE = cluster-role-admin cluster-role-binding cluster-role-view
 CLUSTER_CONFIG_CRD = crd-networkservices crd-networkserviceendpoints crd-networkservicemanagers
 CLUSTER_CONFIG_NAMESPACE = namespace-nsm
-CLUSTER_CONFIGS = $(CLUSTER_CONFIG_ROLE) $(CLUSTER_CONFIG_CRD) $(CLUSTER_CONFIG_NAMESPACE)
+CLUSTER_CONFIGS = $(CLUSTER_CONFIG_ROLE) $(CLUSTER_CONFIG_CRD) $(CLUSTER_CONFIG_NAMESPACE) \
+	nsm-configmap
 
 ifeq ($(NSM_NAMESPACE),)
 NSM_NAMESPACE = `cat "${K8S_CONF_DIR}/${CLUSTER_CONFIG_NAMESPACE}.yaml" | awk '/name:/ {print $$2}'`
 endif
 CONTAINER_REPO?=networkservicemesh
 CONTAINER_TAG?=master
+
+NSMGR_CONTAINERS := nsmd nsmd-k8s nsmdp
+
```

Comment:
Why?

Reply:
to reuse it in k8s-nsmgr-save and k8s-nsmgr-load-images targets